docs/resources/database_grant.md

```diff
@@ -42,6 +42,6 @@ resource snowflake_database_grant grant {
 Import is supported using the following syntax:
 
 ```shell
-# format is database name | privilege | true/false for with_grant_option
-terraform import snowflake_database_grant.example 'databaseName|USAGE|false'
+# format is database name | | | privilege | true/false for with_grant_option
+terraform import snowflake_database_grant.example 'databaseName|||USAGE|false'
```

Comment:
You need to update this file - https://github.com/chanzuckerberg/terraform-provider-snowflake/blob/main/examples/resources/snowflake_database_grant/import.sh 

and then run `make docs`.

Reply:
Ah, sorry. I'll run later.

Reply:
ran at c8f9e2e